controllers/storagecluster/cephcluster.go

```diff
@@ -1454,3 +1419,24 @@ func determineDefaultCephDeviceClass(foundDeviceClasses []rookCephv1.DeviceClass
 	// if no device classes are found in status return empty string
 	return ""
 }
+
+// deleteSucceededPodsWithDuplicateTolerations deletes the succeeded pods of the given app name which have duplicate tolerations
+func (r *StorageClusterReconciler) deleteSucceededPodsWithDuplicateTolerations(appName string, namespace string) error {
```

Comment:
Can you pls pass the key and value both of a label instead of just passing a value?